staging/src/k8s.io/api/authorization/v1/types.go

```diff
@@ -118,6 +118,83 @@ type ResourceAttributes struct {
 	// Name is the name of the resource being requested for a "get" or deleted for a "delete". "" (empty) means all.
 	// +optional
 	Name string `json:"name,omitempty" protobuf:"bytes,7,opt,name=name"`
+	// fieldSelector describes the limitation on access based on field.  It can only limit access, not broaden it.
```

Comment:
nit: indicate on these fields they are alpha level and depend on the AuthorizeWithSelectors feature gate (here and in other type files)